examples/api/mix.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Api.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :api,
+      version: "0.1.0",
+      elixir: "~> 1.18",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      elixirc_paths: ["lib"],
+      aliases: aliases()
+    ]
+  end
+
+  def application do
+    [mod: {Api, []}, extra_applications: [:logger]]
+  end
+
+  defp deps do
+    [
+      {:francis, "~> 0.1"},
+      {:ecto, "~> 3.12"},
+      {:ecto_sql, "~> 3.12"},
+      {:postgrex, "~> 0.20.0"}
+    ]
+  end
+
+  defp aliases do
+    [
+      "ecto.setup": ["ecto.create", "ecto.migrate"],
```

Comment:
A nice follow up would be seeding :raised_back_of_hand:

Reply:
I wanted to avoid having ecto in the mix of francis